wgengine/userspace.go

```diff
@@ -1231,7 +1231,7 @@ func (e *userspaceEngine) Ping(ip netip.Addr, pingType tailcfg.PingType, cb func
 	e.logf("ping(%v): sending %v ping to %v %v ...", ip, pingType, peer.Key.ShortString(), peer.ComputedName)
```

Comment:
Maybe add the ping payload size in this log message? I was running this code without the DF bit patch and had to add that in to help debug.